### DISCUSSION TITLE
Can I slow down sb.cdp.press_keys()

### DISCUSSION THREAD

**Discussion by morndej** [seleniumbase/SeleniumBase#3308] [Q&A] [answered]
"Is there a way to slow down `sb.cdp.press_keys()`? 
I got detected even when using `press_key()`. 
Does it support fine-grained configuration to adjust its speed?

**Comment by mdmintz** [comment, marked as answer]
If these aren't slow enough as is to avoid detection...

```python
sb.cdp.gui_press_key(key)
sb.cdp.gui_press_keys(keys)
```

...then you can create a `for` loop, where you add `sb.sleep(seconds)` to slow things down as much as you want.

**Reply by morndej** [reply]
Thank you for your helpful response!  

I’ll try implementing it to see if it helps avoid detection. Appreciate your guidance!
